### DISCUSSION TITLE
Question

### DISCUSSION THREAD

**Discussion by Drizzle26** [community/community#44968] [Other Feature Feedback, Questions, & Ideas]
How to generate Ssh key using Turmax

**Comment by jge162** [comment]
@Drizzle26 To create an SSH key using terminal (or "tumux"), you can use the following command:


```ssh-keygen -t rsa -b 4096```
This will generate an RSA key with a length of 4096 bits. You will be prompted to choose a location and name for the key, and to enter a passphrase (if you want to add an extra layer of security). Once the key is generated, you can use the command


```ssh-copy-id user@host```
to install the key on the remote host.
replace "user@host" with your correct user and hostname where you want to use the ssh key to connect
Please note that you will need to have the ssh package installed on your computer to use these commands.
